dGame/dComponents/SimplePhysicsComponent.cpp

```diff
@@ -51,6 +51,7 @@ void SimplePhysicsComponent::Serialize(RakNet::BitStream& outBitStream, bool bIs
 	outBitStream.Write(m_DirtyPhysicsMotionState || bIsInitialUpdate);
 	if (m_DirtyPhysicsMotionState || bIsInitialUpdate) {
 		outBitStream.Write<uint32_t>(m_PhysicsMotionState);
+		m_DirtyPhysicsMotionState = false;
```

Comment:
Did you test this? :P

Reply:
nope!

Reply:
![image](https://github.com/user-attachments/assets/ec309292-9382-4adb-89ea-f1c9052b4833)
lgtm!